tests/backends/data/test_fs.py

```diff
@@ -16,7 +16,7 @@
 
 def test_backends_data_fs_data_backend_default_instantiation(monkeypatch, fs):
     """Test the `FSDataBackend` default instantiation."""
-    # pylint: disable=invalid-name
```

Comment:
don't we want this checked ? (invalid-name)

Reply:
You're right!
Adding the `N` set of rules (for `pep8-naming`) but escaping some of them (`N806`, `N815` and `N818`) that could be potentially escaped or corrected in the future.
Note that it does not enforce a minimum length for variables (I fiund that a minimum of 3 is a big constraint), so it won't trigger a warning on `fs` here.

Reply:
Dropping this as it is too restrictive